runtime/resourcedictionary_test.go

```diff
@@ -278,14 +269,14 @@ func TestRuntimeResourceDictionaryValues(t *testing.T) {
 
 	assert.Equal(t,
 		[]string{
-			"3",
-			`"destroying R"`,
 			"3",
 			"4",
 		},
 		loggedMessages,
 	)
 
+	// DestructorTODO: add test for destruction event of R
```

Comment:
Are these TODOs for this PR? It would be good to keep asserting the recursive destruction is performed in the proper order

Reply:
These are addressed in https://github.com/onflow/cadence/pull/2817, once support for default events is finished

Reply:
Nice! Maybe target a separate feature branch for this PR, so we don't forget about these TODOs